services/fdb/client/client.go

```diff
@@ -1361,25 +1365,47 @@ func (r *fdbCLIExcludeCmd) SetFlags(f *flag.FlagSet) {
 
 func (r *fdbCLIExcludeCmd) Execute(ctx context.Context, f *flag.FlagSet, args ...interface{}) subcommands.ExitStatus {
 	req := args[1].(*pb.FDBCLIRequest)
+	// by default the no_wait will be set, unless we explicit set the wait option in command
+	r.req.NoWait = &wrapperspb.BoolValue{
+		Value: true,
+	}
 
-	failed, address := false, false
+	wait, failed, address := false, false, false
+	setOpt := func(opt string, optName string, optValue *bool) (bool, subcommands.ExitStatus) {
```

Comment:
Do we need to return a `bool` too? It seems `subcommands.ExitStatus` can server as an indication of ok/not ok, so  we can return just that.